### DISCUSSION TITLE
Build report output

### DISCUSSION THREAD

**Discussion by martin323232** [inventree/InvenTree#4038] [Q&A]
Can someone please show me where on the build order page I can find the report output button?
Report is enabled in settings as I can output BOM reports.

![image](https://user-images.githubusercontent.com/6439801/206838096-28325bf5-d3e9-45e2-a357-f0cdcfa9837e.png)
![image](https://user-images.githubusercontent.com/6439801/206838211-fab3d16c-281a-447b-ba29-155e3735c2af.png)
![image](https://user-images.githubusercontent.com/6439801/206838227-bd2e36c5-c5e5-4144-8073-100d6ce40174.png)

**Comment by SchrodingersGat** [comment]
Do you have reports enabled?

![image](https://user-images.githubusercontent.com/10080325/206878014-b03f6cda-8fdf-4ec1-8d60-8155a8e5c9ac.png)

**Comment by martin323232** [comment]
Yes, its enabled. I can creat BOM reports
________________________________

**Reply by SchrodingersGat** [reply]
What version of InvenTree are you running? This feature seems to be working just fine on the demo server:

![image](https://user-images.githubusercontent.com/10080325/206898807-3387b6e1-3ade-481a-896a-461bf191852f.png)

**Comment by martin323232** [comment]
Thanks, after a couple of page refreshes it appeared.
I’m having this issue other places also, when allocating stock I need to refresh site for “Untracked stock has not been fully allocated for this Build Order” to change after pressing “Allocate Stock” button
